### ISSUE TITLE
Icons not working properly on demo site

### ISSUE THREAD

**Issue by jaredweiss** [nupic-community/river-view#41]
see http://river-view.herokuapp.com/index.html

The small icons on the buttons (i.e. Details, JSON API, and Source Code) are showing up as boxes rather than the proper icons.

**Comment by rhyolight** [comment]
What browser?

---------
Matt Taylor
OS Community Flag-Bearer
Numenta

**Comment by jaredweiss** [comment]
Chrome version 44.0.2403.107

**Comment by jaredweiss** [comment]
![screen shot 2015-07-28 at 1 24 24 pm](https://cloud.githubusercontent.com/assets/1041441/8942749/0266758e-352c-11e5-952d-c33a5404e755.png)

**Comment by rhyolight** [comment]
I'm on Version 44.0.2403.107 (64-bit) and it looks great. Hey @brev I hate to drag you into this, but bootstrap glyphs aren't working on Jared's Chrome, but they are on mine. Do you know why this might happen?

**Comment by brev** [comment]
broken for me too, I'm seeing the below font+domain error in console: 

> Font from origin 'http://data.numenta.org' has been blocked from loading by Cross-Origin Resource Sharing policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://river-view.herokuapp.com' is therefore not allowed access.

Can't figure out why yours would be working OK, though.

**Comment by jaredweiss** [comment]
interestingly enough, the symbols show up nicely on http://data.numenta.org/index.html

**Comment by jefffohl** [comment]
in the partial, header.tpl, why not use a relative path for scripts, CSS, and such? You can see that they are absolute paths to data.numenta.org.

**Comment by rhyolight** [comment]
Oh, I thought this was a browser issue, but it's just a path issue. 

> in the partial, header.tpl, why not use a relative path for scripts, CSS, and such? You can see that they are absolute paths to data.numenta.org.

Because relative paths must be in relation to the page being rendered (not the partial template), and the partial doesn't know what its path is going to be until runtime.

**Comment by jefffohl** [comment]
so you have {{baseurl}} there in case it is not installed at the root of the web server?

**Comment by rhyolight** [comment]
Yes. It could be installed as an express middleware, in which case "/" will
not point to the application root.

---------
Matt Taylor
OS Community Flag-Bearer
Numenta

**Comment by jefffohl** [comment]
I can implement this middleware. Assign this to me if you would like me to handle it.

**Comment by rhyolight** [comment]
No, there is no middleware necessary to fix this issue. I was just justifying why templates are given a `{{baseurl}}`.
